src/Sylius/Bundle/ShopBundle/Tests/Doctrine/ORM/ProductRepositoryTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ShopBundle\Tests\Doctrine\ORM;
+
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Sylius\Bundle\CoreBundle\Doctrine\ORM\ProductRepository;
+use Sylius\Component\Core\Model\Product;
+
+final class ProductRepositoryTest extends DoctrineRepositoryTestCase
```

Comment:
Could we move this test to `CoreBundle`? It looks like it is not connected with ShopBundle.